pkg/minikube/storageclass/storageclass.go

```diff
@@ -92,6 +76,28 @@ func SetDefaultStorageClass(name string) error {
 			return errors.Wrapf(err, "Error while marking storage class %s as %s", sc.Name, isDefault)
 		}
 	}
-
 	return nil
 }
+
+var getStoragev1 = func() (storagev1.StorageV1Interface, error) {
```

Comment:
why do we have this package variable declared instead of an actual `func`?

Reply:
fixed by refactor.